src/pages/index.js

```diff
@@ -117,6 +148,17 @@ const Page = () => (
             <IndexCardLink to="/supported-open-source/barman/">
               Barman
             </IndexCardLink>
+
+            <IndexCardLink
+              to="/supported-open-source/barman/single-server-streaming/"
+              className="nested-link"
```

Comment:
This is fine for now, but I could see it becoming a property of IndexCardLink, like `<IndexCardLink nested={true}` or similar. Especially if we're going to use this style more